service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -98,6 +105,27 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             .build();
     }
 
+    @PostMapping("/submitted")
+    public void handleSubmittedEvent(@RequestBody @NotNull CallbackRequest callbackRequest) {
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        CaseDetails caseDetailsBefore = callbackRequest.getCaseDetailsBefore();
+        CaseData caseDataBefore = mapper.convertValue(caseDetailsBefore.getData(), CaseData.class);
+
+        List<String> currentDocumentUrls = getBinaryUrlsForNoticeOfPlacementOrder(caseData);
+        List<String> previousDocumentUrls = getBinaryUrlsForNoticeOfPlacementOrder(caseDataBefore);
+        currentDocumentUrls.removeAll(previousDocumentUrls);
+
+        if (!currentDocumentUrls.isEmpty()) {
```

Comment:
[Don't need the if statement](https://github.com/hmcts/fpl-ccd-configuration/pull/863)